addons/mail/static/src/views/activity/activity_controller.scss

```diff
@@ -42,6 +42,18 @@
             margin-right: 16px;
         }
 
+        .o_m2m_avatar, .o_m2m_avatar_empty {
+            width: 32px !important;
+            height: 32px !important;
+            margin-right: 8px;
```

Comment:
Is it really necessary? If yes then for all avatars or just the last one? 🤔

Reply:
yes for all avatars except last one `o_m2m_avatar_empty`.

Reply:
IMO, the space between 2 avatars are sufficient without your margin right. 🤔 
I will check with Alexandra to be sure.

Reply:
After discussing with Alexandra, we can indeed remove that margin right and bottom because IMO, it was more a problem in the activity view definition than a problem for that widget. 🙂

Reply:
Done! Thank you :slightly_smiling_face: